Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -1240,4 +1240,4 @@ public async Task<string> PresignedPutObjectAsync(string bucketName, string obje
             return Tuple.Create(this.restClient.BaseUrl.AbsoluteUri, policy.GetFormData());
         }
     }
-}
+}
```

Comment:
Please remove changes in this file

Reply:
In a previous comment you asked to remove newlines at the end of the file.

Reply:
Will do.